pkg/controllers/node/termination/suite_test.go

```diff
@@ -908,3 +957,13 @@ func ExpectNodeWithNodeClaimDraining(c client.Client, nodeName string) *corev1.N
 	Expect(node.DeletionTimestamp).ToNot(BeNil())
 	return node
 }
+
+func ExpectNotRequeued(result reconcile.Result) {
```

Comment:
These are two functions that I think could be generally useful and I would consider putting in operatorpkg

Reply:
Good call, I've opened a PR: https://github.com/awslabs/operatorpkg/pull/130

Reply:
I'm only just getting back to this, but I think we can defer migration for https://github.com/kubernetes-sigs/karpenter/issues/1348 and continue to use the local version for now. This way we don't have a mix of operatorpkg and local expectations.